daprdocs/content/en/concepts/observability-concept.md

```diff
@@ -4,60 +4,38 @@ title: "Observability"
 
-You can use distributed tracing to help debug and optimize application code. Distributed tracing contains trace spans between the Dapr runtime, Dapr system services, and user apps across process, nodes, network, and security boundaries. It provides a detailed understanding of service invocations (call flows) and service dependencies.
+<img src="/images/observability-tracing.png" width=1000>
 
-Dapr uses [W3C tracing context for distributed tracing]({{<ref w3c-tracing>}})
+### OpenTelemetry collector
+Dapr can also be configured to work with an [OpenTelemtry collector]({{<ref open-telemetry-collector>}}) which offers even more compatibility with external monitoring tools.
 
-It is generally recommended to run Dapr in production with tracing.
+<img src="/images/observability-opentelemetry-collector.png" width=1000>
 
-### Open Telemetry
+### Tracing context
+Dapr uses [W3C tracing]({{<ref w3c-tracing>}}) specification for tracing context and can generate and propagate the context header itself or propagate user provided context headers.
 
-Dapr integrates with [OpenTelemetry](https://opentelemetry.io/) for tracing, metrics and logs. With OpenTelemetry, you can configure various exporters for tracing and metrics based on your environment, whether it is running in the cloud or on-premises.
+## Observability for the Dapr sidecar and system services
+Like for other parts of your system, you will want to be able to observe Dapr itself and collect metrics and logs emitted by the Dapr sidecar that runs along each microservice as well as the Dapr related services in your environment such as the control plane services that are deployed for a Dapr enabled Kubernetes cluster.
 
-#### Next steps
+<img src="/images/observability-sidecar.png" width=1000>
 
-- [How-To: Set up Zipkin]({{< ref zipkin.md >}})
-- [How-To: Set up Application Insights with Open Telemetry Collector]({{< ref open-telemetry-collector.md >}})
+### Logging
+Dapr generates [logs]({{<ref "logs.md">}}) to provide visibility to sidecar operation and to help users identify issues and perform debugging. Logs events contain warning, error, info, and debug messages produced by Dapr system services. Dapr can also be configured to send logs to collectors such as [Fluentd]({{< ref fluentd.md >}}) and [Azure Monitor]({{< ref azure-monitor.md >}}) so they can be easily searched, analyzed and provide insights.
+### Metrics
+Metrics are the series of measured values and counts that are collected and stored over time. [Dapr metrics]({{<ref "metrics">}}) provide monitoring capabilities to understand the behavior of Dapr system services. For example, the service metrics between Dapr sidecars and user apps show call latency, traffic failures, error rates of requests etc. Dapr system services metrics show side car injection failures, health of the system services including CPU usage, number of actor placement made etc.
```

Comment:
this sentence "behavior of Dapr system services" seems to imply that metrics only apply to Dapr system services. The following sentence then says metrics between Dapr and your application. I would make it clearer here what metrics refers to in the 2nd sentence. 

I would also reword this sentence "For example, the service metrics between Dapr sidecar" to "For example, the metrics between a Dapr sidecar and the user application show...". The word "service metrics" here really throws you off. 

It would be good to have a link or even a picture of showing what you mean by the system service metrics.

Reply:
Clarified and added link